Added better behaviour when a queue is missing. (#18)
AnyBlok/anyblok_bus: doc/CHANGES.rst

* Refactored bus console script, Added processes parameter on bus_consumer.
  The goal is to define processes for one queue, by default all the queues 
  are in the same process

> * Added better log when a queue is missing. Now if queues are missing, the 

Reviewer: ```suggestion
* Add better logging when a queue is missing. If a queue is missing, then
```

>   workers won't run

Reviewer: ```suggestion
  workers won't start.
```
